templates/Dockerfile.template

```diff
@@ -0,0 +1,24 @@
+FROM {{ base_image }}
+
+ENV HOME=/home/ops \
+    CONDA_DIR=/opt/conda
+
+USER root
+RUN $CONDA_DIR/bin/conda install -y nb_conda_kernels
+
+USER ops
+RUN $HOME/verdi/bin/pip install 'ipykernel<5.0.0'
+
+RUN cd $HOME \
+    && git clone https://github.com/hysds/notebook_pge_wrapper.git \
+    && cd notebook_pge_wrapper \
+    && $HOME/verdi/bin/pip install -e .
+
+# copy your repo into the docker container
+################################################
+COPY . $HOME/{{ project }}
+################################################
+
```

Comment:
Need to add the running of requirement.ipynb here.

Reply:
i'll handle it in a different PR b/c its in a [separate ticket](https://hysds-core.atlassian.net/browse/HC-353)
but will do